engine/runtime/src/main/java/org/enso/interpreter/runtime/data/text/Text.java

```diff
@@ -93,7 +90,11 @@ public boolean is_normalized() {
       case UNKNOWN -> {
         Normalizer2 normalizer = Normalizer2.getNFDInstance();
         boolean isNormalized = normalizer.isNormalized(toString());
-        setFcdNormalized(isNormalized);
+        if (isNormalized) {
+          fcdNormalized = FcdNormalized.YES;
+        } else {
+          fcdNormalized = FcdNormalized.NO;
+        }
```

Comment:
```suggestion
        fcdNormalized = isNormalized ? FcdNormalized.YES : FcdNormalized.NO;
```

less code = profit

Reply:
Even less code in 4bf1ad79639393baca1d1777700e38148252e2de and this time with 8x times less memory to encode the `fcdNormalized` value.